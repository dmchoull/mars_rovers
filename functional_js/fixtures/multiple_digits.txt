21 21
12 0 N
MMMMMMMMMM
