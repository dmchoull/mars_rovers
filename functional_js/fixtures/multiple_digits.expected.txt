12 10 N
